ckan/templates-midnight-blue/group/snippets/group_item.html

```diff
@@ -1,64 +1,59 @@
 {#
-Renders a media item for a group. This should be used in a list.
+Renders a card for a group. This should be used in a list.
 
 group - A group dict.
 
 Example:
 
-    <ul class="media-grid">
+    <ul class="group-list">
       {% for group in groups %}
         {% snippet "group/snippets/group_item.html", group=group %}
       {% endfor %}
     </ul>
 #}
+
 {% set type = group.type or 'group' %}
 {% set url = h.url_for(type ~ '.read', id=group.name) %}
 {% block item %}
-<li class="media-item">
+<li class="groups-cards">
+  {% block link %}
+  {% set view_label = _('Open details page for {name}').format(name=group.display_name) %}
+    <a href="{{ url }}" aria-label="{{ view_label }}" data-bs-title="{{ view_label }}" data-bs-toggle="tooltip">
   {% block item_inner %}
   {% block image %}
-    <img src="{{ group.image_display_url or h.url_for_static('/base/images/placeholder-group.png') }}" alt="{{ group.name }}" class="media-image img-fluid">
+    <div class="module-image">
+      <img src="{{ group.image_display_url or h.url_for_static('/base/images/placeholder-group.svg') }}" class="img-fluid" alt="{{ group.name }}" />
```

Comment:
I see that the placeholder image for organizations hasn't been replaced. The current theme uses the [building motif](https://demo.ckan.org/organization/) for orgs and a [folder](https://demo.ckan.org/group/) for groups (as in categories). Perhaps this makes more sense and also is worth keeping the same icons for continuity with the current theme